Dockerfile.template

```diff
@@ -32,9 +32,8 @@ RUN apk add --no-cache --virtual .crate-rundeps \
         openjdk8-jre-base \
         python3 \
         openssl \
-        sigar \
-    && apk add --no-cache --virtual .build-deps \
         curl \
+    && apk add --no-cache --virtual .build-deps \
```

Comment:
is there a formatting issue here?

Reply:
`curl` moved from build dependencies to runtime dependencies